### PR TITLE
Add Relevancy Date Interactions

### PR DESCRIPTION
**Pull request by pzalep1**
# Description
This PR adds the ability for a relevancy date to be added to a learning object on the admin dashboard. This also adds a notice below the changelog to indicate when this learning-object will be next reviewed. 

There is also functionality to filter by nextCheck dates on the admin dashboard. 

Screen Shot 2021-08-08 at 3 53 03 PM
Screen Shot 2021-08-08 at 3 53 12 PM
Screen Shot 2021-08-08 at 7 21 21 PM
Screen Shot 2021-08-10 at 1 46 59 PM
Screen Shot 2021-08-10 at 1 47 21 PM

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import 'hammerjs';
```
Comment (mfranz1) [inline src/main.ts:1]: Is this needed?

**Review by mfranz1 — COMMENTED** [review]
Just a comment

File: src/main.ts (hunk 1 above)
Comment (pzalep1) [reply src/main.ts:1]: Yes, it's required for angular material.

**Review by mfranz1 — APPROVED** [review]

**Review by Cwagne17 — APPROVED** [review]

File: src/app/cube/details/details.component.html [hunk 2]
```diff
@@ -97,6 +97,7 @@ <h3 class='section-title'>Authors</h3>
             <div class='details__content-container__version-card-container'>
               <clark-version-card [version]='learningObject.version' [showButton]='loggedin && auth.user.emailVerified' (clickButtonEvent)='openViewAllChangelogsModal()'></clark-version-card>
             </div>
+            <p class="relevancy__notice">This learning object will be next reviewed by {{learningObject?.nextCheck | date:'mediumDate'}}</p>
```
Comment (EVogel1999) [inline src/app/cube/details/details.component.html:100]:
```suggestion
            <p class="relevancy__notice">This learning object will be reviewed for relevancy next by {{learningObject?.nextCheck | date:'mediumDate'}}</p>
```

**Review by EVogel1999 — APPROVED** [review]
One small suggestion

**Review by jmulholland4014 — COMMENTED** [review]
Is there functionality to stop the user from setting a date in the past?

**Comment by pzalep1** [comment]
> Is there functionality to stop the user from setting a date in the past?

Yes, the calendar does not allow for anything less than 4 months in the future from the current day.

**Review by jmulholland4014 — APPROVED** [review]

**Review by EVogel1999 — APPROVED** [review]
Git Merge

**Review by Cwagne17 — APPROVED** [review]

**Pull request merged**